.changeset/silly-rocks-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/chip": major
+---
+
+Chip storyboard missing some import
```

Comment:
@kuri-sun as this change only affects the `storybook` we do not need to add a changeset because the stories are not added into the package once released